### DISCUSSION TITLE
Transfer a bot chat to pending chat when agent is offline

### DISCUSSION THREAD

**Discussion by Broninn** [LiveHelperChat/livehelperchat#2130] [Q&A]
Hello everyone, would there be any configuration for the case below:

I have a customer service bot, and if the user wishes, they can speak to an attendant. Up to this point everything is OK. I also created a trigger that allows the user to report their problem if the attendant is offline. Now comes the question, transfer this chat to pending chat.

Here's my last trigger, where everything is ok so far, the only thing missing is this transfer to pending chat setting. I'm on version 4.39

![image](https://github.com/user-attachments/assets/112eefbc-caa6-4d71-8178-ab729deb64d3)

**Comment by remdex** [comment]
https://doc.livehelperchat.com/docs/bot/bot-update-current-chat#stop-chat-and-transfer-to-human
